src/test/tests.ts

```diff
@@ -809,15 +805,16 @@ describe('Server', function () {
 
     setTimeout(() => {
       expect(connectionCallbackSpy.calledOnce).to.be.true;
-      expect(connectionCallbackSpy.getCall(0).args[0]).to.equal('Error');
+      expect(connectionCallbackSpy.getCall(0).args[0].message).to.equal('Error');
```

Comment:
Just when handling GQL_CONNECTION_ERROR you want to make sure the legacy packet looks exactly the same so this test should be the  exact same if forceLegacy is used